spec/support/features/batch_edit_actions.rb

```diff
@@ -15,10 +15,12 @@ def fill_in_batch_edit_fields_and_verify!
       fill_in "generic_work_#{field_id}", with: "NEW #{field_id}"
 
       find("##{field_id}_save").click
+
+      ajax_wait(15)
```

Comment:
Aw, this is a bummer. Do we know if this is still true? I won't block on this, but maybe a ticket to investigate this?

Reply:
Yes in dassie with use_valkyrie=true. Integration tests like https://github.com/samvera/hyrax/pull/6148/files#diff-7eea01c84c5eba32121b213d71b5dd9f86b7968181dd4a273348d42e00463bd3R65 will failed with the original 5 seconds. A think this is dynamic wait time that could exit the loop when ajax call response back, which could be a better pattern than the code in https://github.com/samvera/hyrax/pull/6148/files#diff-5522864e208680e8d4491bd348df8c440287c7b89c1efd4e36c4d97b316300c6L21 that waits for constant 5 seconds for every test.